pkg/project/version_bundle.go

```diff
@@ -15,11 +15,125 @@ func VersionBundle(p string) versionbundle.Bundle {
 					"https://github.com/giantswarm/cluster-operator/pull/888",
 				},
 			},
+			{
+				Component:   "chart-operator",
+				Description: "Adjust ClusterRole permissions.",
+				Kind:        versionbundle.KindChanged,
+				URLs: []string{
+					"https://github.com/giantswarm/chart-operator/releases/tag/v0.11.3",
+				},
+			},
+			{
+				Component:   "cert-manager",
+				Description: "Improve helm chart for clusters with restrictive network policies.",
+				Kind:        versionbundle.KindChanged,
+				URLs: []string{
+					"https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.4",
+				},
+			},
+			{
+				Component:   "cert-manager",
+				Description: "Update manifests for Kubernetes 1.16 compatibility.",
+				Kind:        versionbundle.KindChanged,
+				URLs: []string{
+					"https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.3",
+				},
+			},
+			{
+				Component:   "kiam",
+				Description: "Improve helm chart for clusters with restrictive network policies.",
+				Kind:        versionbundle.KindChanged,
+				URLs: []string{
+					"https://github.com/giantswarm/kiam-app/releases/tag/v1.0.3",
+				},
+			},
+			{
+				Component:   "kiam",
+				Description: "Update manifests for Kubernetes 1.16 compatibility.",
+				Kind:        versionbundle.KindChanged,
+				URLs: []string{
+					"https://github.com/giantswarm/kiam-app/releases/tag/v1.0.2",
```

Comment:
```suggestion
					"https://github.com/giantswarm/kiam-app/releases/tag/v1.0.4",
```

Could we use 1.0.4? There are a couple of fixes that I think we need. https://github.com/giantswarm/kiam-app/pull/12 https://github.com/giantswarm/kiam-app/pull/11